.circleci/config.yml

```diff
@@ -1,120 +1,108 @@
 version: 2.1
 
-# add your orb below, to be used in integration tests (note: a @dev:alpha
-# release must exist; you'll need to publish manually at least once)
-# you can use scripts/publish-alpha.sh to publish a @dev:alpha release
 orbs:
-  codacy: codacy/base@dev:alpha
-  orb-tools: circleci/orb-tools@8.5.0
+  codacy: codacy/base@0.0.13
+
+references:
```

Comment:
do you need to deliver the changes for this file in this PR?

Reply:
Nope, this was just based on top of the circleci branch. Now that it is merged I'll rebase on master and these changes will disappear.